addons/web/tooling/hooks/pre-commit

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash
+current_git=$(git rev-parse --show-toplevel)
 tooling_dir=$(cd -- "$(dirname "$0")" &> /dev/null && cd .. && pwd)
 if ! cmp -s -- "$tooling_dir/_package.json" package.json; then
     echo "Your package.json is out of date, reloading the tooling using the reload script"
-    "$tooling_dir/reload.sh"
+    python3 "$tooling_dir"/enable.py nodejs remove --force-node --addons-path=$current_git
+    python3 "$tooling_dir"/enable.py nodejs install --force-node --addons-path=$current_git
```

Comment:
How cross-platform is this? Does this work on mac?

Reply:
yes, that's the step i need to test :D

Reply:
also, I'm not sure about the version conflict thing because the pre-commit points to the relevant sript.
Be in 18: install everything: pre-commiit is the one in this pr
Checkout 16: git's hook still point to hooks/pre-commit, which has been checked out
commit: reload, refresh, enable exist in their old form.

Does that make sense ?

Reply:
on Macos ventura 13.4 
- python3 is an actual command and according to @bastien-ipb is the default executable to python when you install python on Macos
- operations that copy working well.
- precommit working well too
(Modulo that I tested script execution in some very basic setting.....)

@sdegueldre

Reply:
> also, I'm not sure about the version conflict thing because the pre-commit points to the relevant sript. Be in 18: install everything: pre-commiit is the one in this pr Checkout 16: git's hook still point to hooks/pre-commit, which has been checked out commit: reload, refresh, enable exist in their old form.
> 
> Does that make sense ?

It does, I forgot that we don't copy the hooks to the git hooks folder but change the hoosPath instead. Still worried about having different versions of the tooling doing different things.